src/tools/rbd_mirror/LeaderWatcher.h

```diff
@@ -98,6 +98,8 @@ class LeaderWatcher : protected librbd::Watcher {
         watcher(watcher) {
     }
 
+    virtual ~LeaderLock() = default;
```

Comment:
@adamemerson `LeaderLock` derives from `librbd::ManagedLock<ImageCtxT>`, and the latter does have a virtual dtor. so `LeaderLock` will have a virtual dtor even if we don't put
```c++
virtual ~LeaderLock() = default;
```
here, am i right?